app/core/db.py

```
@@ -0,0 +1,35 @@
+    pool_pre_ping=True,
+)
+
+session_maker = async_sessionmaker(
+    bind=engine,
+    class_=AsyncSession,
+    expire_on_commit=False,
+    autoflush=False,
+)
+
+
+async def init_db():
```

Ruff (0.12.2)

Missing return type annotation for public function `init_db`

Add return type annotation: `None`

(ANN201)